data/json/items/comestibles/alcohol.json

```diff
@@ -1060,19 +1062,20 @@
     "quench": 8,
     "healthy": 1,
     "addiction_potential": 1,
-    "calories": 130,
+    "calories": 541,
     "description": "Droplet shaped ant food.  It's like a thick balloon the size of a baseball, filled with sticky liquid.  Unlike bee honey, this has mostly a sour taste, probably because ants feed upon a variety of things.",
     "price": 1000,
     "material": [ "junk", "honey" ],
     "volume": 1,
+    "charges": 2,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.

Reply:
i don't know about that, for this item. it still gets you just as drunk.